debug = false
trace = false

server.port=8080

## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)

# Docker
#spring.datasource.url = jdbc:postgresql://postgres:5432/postgres
#spring.datasource.username = postgres
#spring.datasource.password = password

# Work
spring.datasource.url = jdbc:postgresql://localhost:5432/postgres

# Home
#spring.datasource.url = jdbc:postgresql://localhost:5432/useraccount
#spring.datasource.username = developer
#spring.datasource.password = admin

# Hibernate

# Hibernate ddl auto (create, create-drop, validate, update)
spring.jpa.hibernate.ddl-auto = update

logging.level.org.hibernate.SQL = DEBUG
logging.level.org.hibernate.type = TRACE

spring.flyway.baseline-on-migrate = true

# I encountered a bug in Hibernate and this was the solution
# https://stackoverflow.com/questions/49680199/java-lang-reflect-invocationtargetexception-null
spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false